resources/views/profile/edit.blade.php

```diff
@@ -0,0 +1,29 @@
+@extends('layouts.layout')
+
```

Comment:
Voor consistentie is het denk ik beter om alles in de Nederlandse taal te tonen

Reply:
fixed